test/lib/jdk/test/lib/Asserts.java

```diff
@@ -234,59 +234,58 @@ public static void assertSame(Object lhs, Object rhs, String msg) {
     }
 
     /**
-     * Asserts that {@code lhs} is the same byte array as {@code rhs}.
+     * Asserts that {@code actual} has the same content as {@code expected}.
      *
-     * @param lhs The left hand side of the comparison.
-     * @param rhs The right hand side of the comparison.
+     * @param expected The expected value
+     * @param actual The actual value
      * @throws RuntimeException if the assertion is not true.
      * @see #assertEqualsByteArray(byte[], byte[], String)
      */
-    public static void assertEqualsByteArray(byte[] lhs, byte[] rhs) {
-        assertEqualsByteArray(lhs, rhs, null);
+    public static void assertEqualsByteArray(byte[] expected, byte[] actual) {
+        assertEqualsByteArray(expected, actual, null);
     }
 
     /**
-     * Asserts that {@code lhs} is not the same byte array as {@code rhs}.
+     * Asserts that {@code actual} does not have the same content as {@code unexpected}.
      *
-     * @param lhs The left hand side of the comparison.
-     * @param rhs The right hand side of the comparison.
+     * @param unexpected The unexpected value
+     * @param actual The actual value
      * @throws RuntimeException if the assertion is not true.
      * @see #assertNotEqualsByteArray(byte[], byte[], String)
      */
-    public static void assertNotEqualsByteArray(byte[] lhs, byte[] rhs) {
-        assertNotEqualsByteArray(lhs, rhs, null);
+    public static void assertNotEqualsByteArray(byte[] unexpected, byte[] actual) {
```

Comment:
For inequality, would "expectedNot" or "targetValue" better than "unexpected"? Or is there similar wording used elsewhere that you are basing this on? This can generally be replaced with `!assertEqualsByteArray(...)`. If you use "targetValue", this is more neutral name for arguments. Method name indicates whether the check is for equality or inequality.

Reply:
FWIW, JUnit uses `unexpected`, testNG uses `expected` in assertNotEquals.

Reply:
Thanks Dan. `unexpected` seems a simpler and straightforward name for me.